source/adminguide/virtual_machines/importing_vmware_vms_into_kvm.rst

```diff
@@ -0,0 +1,118 @@
+
+|import-vm-from-vmware-to-kvm.png|
+
+Selecting the Destination cluster
+---------------------------------
+
+CloudStack administrators must select a KVM cluster to import the VMware Virtual Machines (left side of the image above). Once a KVM cluster is selected, the VMware Datacenter selection part is displayed (right side of the image above).
+
+Selecting the VM from a VMware Datacenter
+-----------------------------------------
+
+CloudStack administrators must select the Source VMware Datacenter:
+
+    - Existing: The existing zones are listed, and for each zone CloudStack will list if there is any VMware Datacenter associated to it. In case it is, it can be selected
+    - External: CloudStack allows listing Virtual Machines from a VMware Datacenter that is not associated to any CloudStack zone. To do so, it needs the vCenter IP address, the datacenter name, and username and password credentials to log in the vCenter.
+
+Once the VMware Datacenter is selected, click on List VMware Instances to display the list of Virtual Machines on the Datacenter
+
+
+Converting and importing a VMware VM
+------------------------------------
+
+.. note:: CloudStack allows importing Running Linux Virtual Machines, but it is recommended that the Virtual Machine to import is powered off and has been gracefully shutdown before the process starts. For Windows Virtual Machines, it is not possible to import them while running.
+
+When importing a Virtual Machine from VMware to KVM, CloudStack performs the following actions:
+
+    - Cloning the Source Virtual Machine on the selected VMware Datacenter: The source Virtual Machine will be cloned in the original state (running or stopped for Linux VMs, or stopped for Windows VMs). The recommended state is the stopped state to prevent data inconsistencies or loss when cloning the virtual machine.
+    - Converting the Cloned Virtual Machine to KVM using virt-v2v: CloudStack (or the administrator) selects a running and Enabled KVM host to perform the conversion from VMware to KVM using virt-v2v. If the binary is not installed, then the host will fail the migration. In case it is installed it will perform the conversion into a temporary location (which can be selected by the administrator) to store the converted QCOW2 disks of the virtual machine. The disks are then moved into the destination storage pools for the virtual machine. The conversion is a long-lasting process which can be set to time out by the global setting 'convert.vmware.instance.to.kvm.timeout'. The conversion processes take a long time because virt-v2v creates a temporary virtual machine to inspect the source VM and generate the converted disks with the correct drivers. Additionally, it needs to copy the converted disks into the temporary location.
+
+.. note:: Please consider not restarting the management servers during the imports since this action can cause failures on the on-going importing processes.
```

Comment:
```suggestion
.. note:: Please consider not restarting the management servers while importing as it will lead to the interruption of the process and you will need to start again.
```